### PR TITLE
issue-434

### PR DESCRIPTION
**Pull request by hingham**

### CODE REVIEW

**Comment by hingham** [comment]
Tasks completed:
- [x] Deleted one repeated svg file
- [x] updated the file paths for the logo images in the 2018 folder

This is my first PR for this repo. Let me know if there is anything you would like changed or updated!

File: packages/2018/src/assets/LandingPage.js [hunk 1]
```diff
@@ -7,14 +7,17 @@ import DataList from './DataList';
...
-import logo from '../../assets/civic-logo-animated.svg';
-import smallLogo from '../../assets/civic-logo-c.svg';
-import brain from '../../assets/brain.svg';
-import brainMobile from '../../assets/brain-mobile.svg';
-import trees from '../../assets/trees.png';
-import hackOregonLogo from '../../assets/hack-oregon-logo.png';
-import heartMail from '../../assets/heartmail.png';
-import twitterLogo from '../../assets/twitter-logo.png';
+import logo from '../../../component-library/assets/civic-logo-animated.svg'
+import smallLogo from '../../../component-library/assets/civic-logo-c.svg'
```
[the remark this answers is not in the document]
Comment (hingham) [reply packages/2018/src/assets/LandingPage.js]: Great! It looks like smallTogo is type squareLogo, so I replaced it with:
<Logo type="squareLogo"/>

File: packages/2018/src/assets/LandingPage.js [hunk 2]
```diff
@@ -7,14 +7,17 @@ import DataList from './DataList';
...
+import brain from '../../../component-library/assets/brain.svg'
```
[the remark this answers is not in the document]
Comment (hingham) [reply packages/2018/src/assets/LandingPage.js]: I believe these are now all updated

File: packages/2018/src/components/PortlandCollectionPage.js [hunk 3]
```diff
@@ -135,7 +135,7 @@ const PortlandCollectionPage = () => (
     <div className={leftContainer}>
       <div className={homeLink}>
         <Link to="/">
-          <img src={require(`../assets/civic-logo-animated.svg`)} />
+          <img src={require(`../../../component-library/assets/civic-logo-animated.svg`)} />
```
[the remark this answers is not in the document]
Comment (hingham) [reply packages/2018/src/components/PortlandCollectionPage.js]: Updated this as well

**Comment by jaronheard** [comment]
@hingham Looks great! Good to merge!

**Pull request merged**